### PR TITLE
PIL-1299 Tiered Instabug Support Levels

### PR DESCRIPTION
**Pull request by divyaBacancy**
emailsupport

livechat

### CODE REVIEW

File: src/screens/Menu/Menu/Menu.js [hunk 1]
```diff
@@ -108,10 +121,21 @@ const Menu = () => {
           onPress={goToSettings}
         />
         <MenuItem title={t('item.addressBook')} icon="contacts" onPress={goToInviteFriends} />
-        <MenuItem title={t('item.supportChat')} icon="message" onPress={goToSupportChat} />
+        {isPlrBalanceZero && <MenuItem title={t('item.supportChat')} icon="message" onPress={goToSupportChat} />}
```
Comment (IAmKio) [inline src/screens/Menu/Menu/Menu.js:124]: If their PLR balance is zero, they should only get email support.

File: src/screens/Menu/Menu/Menu.js [hunk 2]
```diff
@@ -108,10 +121,21 @@ const Menu = () => {
...
         {repliesFlag && (
           <MenuItem title={t('item.supportConversations')} icon="message" onPress={goToSupportConversations} />
         )}
+        {!isPlrBalanceZero &&
+          (enoughPlrBalance ? (
```
Comment (IAmKio) [inline src/screens/Menu/Menu/Menu.js:129]: Is just `enoughPlrBalance` enough maybe?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few minor questions / changes - other than that - it looks good!

File: src/screens/Menu/Menu/Menu.js [hunk 3]
```diff
@@ -108,10 +121,21 @@ const Menu = () => {
           onPress={goToSettings}
         />
         <MenuItem title={t('item.addressBook')} icon="contacts" onPress={goToInviteFriends} />
-        <MenuItem title={t('item.supportChat')} icon="message" onPress={goToSupportChat} />
+        {isPlrBalanceZero && <MenuItem title={t('item.supportChat')} icon="message" onPress={goToSupportChat} />}
         {repliesFlag && (
           <MenuItem title={t('item.supportConversations')} icon="message" onPress={goToSupportConversations} />
         )}
```
[the remark this answers is not in the document]
Comment (divyaBacancy) [reply src/screens/Menu/Menu/Menu.js]: Done

**Review by IAmKio — APPROVED** [review]
Great work - thanks! 🎉

**Pull request merged**